README.rst

```diff
@@ -224,3 +224,9 @@ and with the financial support from Région Ile de France and Confiance.ai.
 ==========
 
 MAPIE is free and open-source software licensed under the `3-clause BSD license <https://github.com/simai-ml/MAPIE/blob/master/LICENSE>`_.
+
+
+📚 Citation
+===========
+
+If you use MAPIE in your research, please cite using `citations file <https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files>`_ on our repository.
```

Comment:
```suggestion
If you use MAPIE in your research, please cite using `citations file <CITATION.CFF>`_ on our repository.
```

Reply:
Check that this change works.

Reply:
I'm not sure this is useful as it does not tell you how to use the `CITATION.CFF file`.
But we can link to that anyways.